src/django/api/filters/submission.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework.filters import BaseFilterBackend
+
+from ..models.role import Roles
+
+
+class UserSubmissionFilterBackend(BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        if request.user.role == Roles.CONTRIBUTOR:
+            queryset.filter(boundary__utility__in=request.user.utilities)
```

Comment:
Sorry for the piecemeal review. Other things I noticed while working on the ReferenceImage serializer.

m2m relationships need all() to get a queryset, otherwise `.utilities` is just a manager (and isn't iterable)
```suggestion
            queryset = queryset.filter(boundary__utility__in=request.user.utilities.all())
```